src/plugins/so_management/kibana.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Should CODEOWNERS be updated to give this plugin an owner?

Reply:
Good catch. updated